### PR TITLE
New backoffice: Fix a few glitches in the dashboard tutorial

### PR DESCRIPTION
**Pull request by kjac**
## Description

The multi-part dashboard tutorial for the new backoffice has a few glitches and errors. While these are somewhat simple to fix, they do pose an unnecessary barrier to get started.

While I was there, I also opted to use "part" instead of "session" when referring to the different parts of the tutorial, and cleaned up a few wordings.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V14

## Deadline (if relevant)

Not relevant 😄

### CODE REVIEW

File: new-backoffice/tutorials/creating-a-custom-dashboard.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This page is a work in progress. It will be updated as the software evolves.
 
 This guide takes you through the steps to set up a Custom Dashboard in Umbraco.
 
-The steps we will go through session 1 are:
+The guide is divided into several parts. In this first part will go through:
```
Comment (jonat123) [inline new-backoffice/tutorials/creating-a-custom-dashboard.md:15]:
```suggestion
The guide is divided into four parts. The first part will go through the following:
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@kjac Looks really good!

Just made a small suggestion to fix the only vale warning :-)

**Comment by kjac** [comment]
Thanks @jonat123 - all ready to go then 😄

**Review by jonat123 — APPROVED** [review]
@kjac Awesome!

Looks good :-) 

Will merge in :D

**Pull request merged**